src/resources/views/crud/fields/relationship/fetch_or_create.blade.php

```diff
@@ -316,14 +316,14 @@ function setupInlineCreateButtons(element) {
 
             },
             error: function (result) {
-                // Show an alert with the result
-                swal({
-                    title: "error",
-                    text: "error",
-                    icon: "error",
-                    timer: 4000,
-                    buttons: false,
-                });
+                @if(!config('app.debug'))
+                new Noty({
+                    type: "error",
+                    text: "<strong>{{ trans('backpack::crud.ajax_error_title') }}</strong><br>{{ trans('backpack::crud.ajax_error_text') }}"
+                }).show();
+                @endif
```

Comment:
Ugh... let's not put PHP inside JS inside BLADE. We might move all field JS to `.js` files in the future, no point in making it difficult for our future selves. 

I think it'd better to use an attribute on the field, say `data-debug`, that we then use in JS to show a noty error or not.

Reply:
@tabacitu in fact, I added this but I had some doubts about it.
This is here to avoid showing the Noty notification on debug, asking to refresh the page.
The error frame will appear in debug mode only, the Noty notification can appear in both maybe ...

Maybe there is no need for this extra attribute on JS. Let me know what you think.

1) Remove the validation and the notification will appear with the error frame.
2) Create the attribute on JS to show the error frame only without the notification.

Reply:
Moved it to `data-debug` 👌